src/jit/lsra.cpp

```diff
@@ -8581,20 +8570,16 @@ void LinearScan::insertMove(
     // This var can't be marked lvRegister now
     varDsc->lvRegNum = REG_STK;
 
-    var_types lclTyp = varDsc->TypeGet();
-    if (varDsc->lvNormalizeOnStore())
-    {
-        lclTyp = genActualType(lclTyp);
-    }
+    var_types lclTyp = genActualType(varDsc->TypeGet());
```

Comment:
For this case, I think you need to use the original type ("actual" type always seems a bit of a misnomer to me) if `fromReg` is `REG_STK`. Which puts you down that path again of having to worry about byteable registers, right?